simapp/go.mod

```
@@ -234,6 +234,8 @@ replace cosmossdk.io/x/upgrade => ../x/upgrade
 
 // Below are the long-lived replace of the SimApp
 replace (
+	// use cosmos fork of api
```

I thought the api go module in this repo was the Celestia fork so I don't understand why Cosmos is used in this comment.

```suggestion
	// use Celestia fork of api
```

---

I had phrased as a fork of cosmos i.e. cosmos fork but I totally get calling it a celestia fork as well. Just wasn't putting too much thought into it